src/main/java/io/jsonwebtoken/JwtParser.java

```diff
@@ -125,6 +125,15 @@
     JwtParser require(String claimName, Object value);
 
     /**
+     * Replaces the system time-of-day call to return this fixed value instead. A {@code null} value will remove
+     * the fixed date output and return the system time-of-day instead.
+     *
+     * @param now the time-of-day to use for parsimg this JWT or {@code null} to use the current time-of-day
+     * @return the builder instance for method chaining.
+     */
+    JwtParser setFixedClock(Date now);
```

Comment:
Per the wording of your own issue subject line, I think it'd be better/more robust if the user can inject their own clock:

```java
JwtParser setClock(Clock clock) {
    this.clock = clock;
}
```

Where Clock looks like this:

```java
public interface Clock {
    Date now();
}
```

And you could have a default implementation:

```java
public class FixedClock implements Clock {

    private final Date date;

    public FixedClock() {
        this(new Date());
    }
    public FixedClock(Date date) {
        this.date = date;
    }

    public Date now() {
        return this.date;
    }
}
```

And then you could call:

```java
Jwts.parser().setClock(new FixedClock(new Date()));
```

Allows for more extensible implementations and testing scenarios.